ciphey/basemods/Decoders/base69.py

```diff
@@ -0,0 +1,81 @@
+# community
```

Comment:
You are no longer community so don't include this + your github repo :) 

This was purely so we could track down who wrote the original code easily && for our tag system that's coming soon:tm: